app/services/migration/migrators/application.rb

```diff
@@ -25,6 +25,8 @@ class Application < Base
       teacher_catchment_country
       notes
       funded_place
+      created_at
```

Comment:
Here and in user, are we making sure this is covered?
> Overwrite updated_at with the one from ECF, or set to the date of migration if anything changed from ECF

Reply:
`updated_at` will be set to Time.now when:
- Application `ukprn` is changed from ECF
- User `trn` and `email` is changed from ECF